cypress/integration/App.e2e.js

```diff
@@ -0,0 +1,49 @@
+describe('App End-to-end Test', () => {
+  it('test app', () => {
```

Comment:
I'd suggest splitting into smaller tests and updating test description to be more descriptive.

Reply:
Splitted into smaller tests.